components/EventItem.tsx

```diff
@@ -51,50 +48,63 @@ const EventItem = ({ event }: Props) => {
           <Text style={styles.titleText}>{event.title}</Text>
 
           <View style={[styles.location, styles.flexRow]}>
-            <Icon name="location-on" color={event.colorHex} />
+            <MaterialCommunityIcons
```

Comment:
We may want these icons to also grow with the font size @mhayes853 
Also we can put the icon within the Text component so it appears as inline, in case the text wraps on larger font sizes.

Reply:
When it comes to icons next to text I've been using 8px margins between the 2, so we should probably just do that here as well for consistency.

And we can deal with abnormally large font sizes in a separate PR.